skills/csharp/experimental/itsmskill/Dialogs/MainDialog.cs

```diff
@@ -230,9 +232,15 @@ private async Task<DialogTurnResult> RouteStepAsync(WaterfallStepContext stepCon
                 if (luisService != null)
                 {
                     var result = stepContext.Context.TurnState.Get<ITSMLuis>(StateProperties.ITSMLuisResult);
-                    var intent = result?.TopIntent().intent;
+                    var (intent, score) = result.TopIntent();
```

Comment:
Do we need to check "result == null" here?

Reply:
No, if luisService is not null, ITSMLuis will be set in OnBeginDialogAsync or OnContinueDialogAsync.